src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -225,4 +229,34 @@ private static string GetValidFileName(string fileName)
         fileName = Uri.EscapeDataString(fileName.AsFileName(false));
         return fileName;
     }
+
+    private static string GetFooterContent(Instance instance)
+    {
+        TimeZoneInfo norwegianTimeZone = TimeZoneInfo.FindSystemTimeZoneById("Europe/Oslo");
```

Comment:
Haven you tested this in a container also? I hope base images that apps use have the tz db/`tzdata` installed, that's not always the case and depends on the base image used. We use alpine and in some versions `tzdata` had to be manually installed. So it would be nice to get this tested with a fully built app

Reply:
I managed to test this, so it looks fine
![image](https://github.com/user-attachments/assets/01bc2c9e-346e-41fd-8589-d739675b8c66)
![image](https://github.com/user-attachments/assets/479e3185-1a2d-4cfc-a984-0e6ee86b62dc)

Reply:
![image](https://github.com/user-attachments/assets/39ad9b9e-6f4e-40a6-9a24-ae9d070d5477)

:(

Reply:
Looks like we would need `RUN apk add --no-cache icu-libs tzdata` to make this work